src/components/Boxes/NotificationPopUp.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import { string, object, func, element } from 'prop-types';
+import LinkHandler from '../Link';
+import styled, { keyframes } from 'styled-components';
+
+const NotificationPopUp = ({
+  title,
+  icon,
+  time,
+  description,
+  to,
+  onClick = () => {},
+  className,
+  otherProps
+}) => {
+  return (
+    <Container className={className} {...otherProps} onClick={onClick} to={to}>
+      {icon && (
+        <IconContainer>
+          <IconCircle>
+            {icon}
+          </IconCircle>
+        </IconContainer>
+      )}
+      <HeaderContentContainer>
+        <Title>
+          {title}
+        </Title>
+        <Description>
+          {description}
+        </Description>
+      </HeaderContentContainer>
+      <TimeLeft>
+        {time}
+      </TimeLeft>
+    </Container>
+  );
+};
+
+const FadeIn = keyframes`
+  from {
+    opacity: 0;
+  }
+
+  to {
+    opacity: 1;
+  }
+`;
```

Comment:
Wat is het nut ervan om hier een animation te gebruiken als het enkel de opacity aanpast? Kunt ge dan niet `transition: opacity 200ms ease` gebruiken?

Reply:
een gewone transition doet geen fade in on mount. De animation doet dit wel